lib/form_data.js

```diff
@@ -192,7 +210,8 @@ FormData.prototype._getContentDisposition = function(value, options) {
 
   // custom filename takes precedence
   // fs- and request- streams have path property
-  var filename = options.filename || value.path;
+  // formidable and the browser add a name property.
+  var filename = options.filename || value.name || value.path;
```

Comment:
This is the one related to #163 right? 

I don't remember what we decided on `browser adds name property`, since browser doesn't really interact with form-data.